cnxdb/migrations/20170911201035_transform_cnxml_to_html.py

```diff
@@ -38,16 +38,19 @@ def get_outdated_baked_books(cursor, limit=''):
     version = rhaptos.cnxmlutils.__version__
     version_text = '%data-cnxml-to-html-ver="{}"%'.format(version)
     logger.info('Looking for baked html without {}'.format(version_text))
+    # Only look for latest books authored by OpenStaxCollege
     cursor.execute("""\
 WITH collated_files AS (
     SELECT file, context
         FROM collated_file_associations c
         NATURAL JOIN files
         JOIN modules m ON c.item = m.module_ident
     WHERE m.portal_type = 'Module'
+      AND m.authors @> '{{"OpenStaxCollege"}}'
```

Comment:
I don't know that this is sufficient in finding all "OpenStax" books. But let's roll with it.

Reply:
This is what Ryan (I think) suggested in a call when I brought up the problem of rebaking all the books automatically.  Apparently on production, all the openstax books are by the same author?